torch/serialization.py

```diff
@@ -1159,7 +1160,7 @@ def load(
     map_location: MAP_LOCATION = None,
     pickle_module: Any = None,
     *,
-    weights_only: Optional[bool] = None,
+    weights_only: Optional[bool] = not IS_FBCODE,
```

Comment:
if you are changing it to hard boolean, please remove `Optional` and delete the logic associated with opt-in checks
```suggestion
    weights_only: bool = not IS_FBCODE,
```